.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - 1.32.0
+  - 1.36.0
```

Comment:
I'm pretty happy with moving the required version up. Going above `1.34` unlocks `TryFrom` ;)